plugins/dell-k2/fu-dell-k2-ec.c

```diff
@@ -44,12 +44,14 @@ typedef struct __attribute__((packed)) { /* nocheck:blocked */
 	guint32 dock_status;
 	guint16 dock_state;
 	guint16 dock_config;
-	guint8 dock_mac_addr[48];
+	guint8 dock_mac_addr[6];
```

Comment:
with this change does the struct need a new version associated with a specific firmware?  If so; I think it's a stronger reason to split out to rustgen _now_.  Have a rustgen definition for v1 and another for v2 etc.

Reply:
The total size remain unchanged, so it's a bugfix for wrong length used for the addr.

Reply:
ahh I see.